net/advanced-operations/compare-pdf-documents/_index.md

```diff
@@ -11,21 +11,23 @@ sitemap:
     priority: 0.7
 ---
 
-## Comparing PDF Documents with Aspose.PDF for .NET
+# Comparing PDF Documents with Aspose.PDF for .NET
 
 When working with PDF documents, there are times when you need to compare the content of two documents to identify differences. The Aspose.PDF for .NET library provides a powerful toolset for this purpose. In this article, we'll explore how to compare PDF documents using a couple of simple code snippets.
 
 The comparison functionality in Aspose.PDF allows you to compare two PDF documents page by page. You can choose to compare either specific pages or entire documents. The resulting comparison document highlights differences, making it easier to identify changes between the two files.
```

Comment:
Please rewrite and mention all ways to compare PDFs. A list would be better, don't you think?